public/css/style.css

```diff
@@ -36,11 +48,14 @@ svg {
   margin: 0;
 }
 
-svg {
-  display: block;
-  position: absolute;
-  top: 30%;
-  left: 5%;
-  width: 90%;
-  height: 70%;
+/* Main page menu button */
+#burger {
+  margin-top: 0.7em;
+  width:2.5em;
+  height:2.5em;
+}
+
+#mySideMenu {
+  /* transition: all 0.5s ease-in; */
```

Comment:
Get rid of the commented out line but apart from that, I'm happy

Reply:
oups missed that one! haha... getting rid of it now :) Thanks Claire!